uniswap/model/parts/uniswap_model.py

```diff
@@ -18,16 +18,41 @@ def p_actionDecoder(_params, substep, sH, s):
         'UNI_burn': 0,        
     }
     
+    # Token prices
+    model_Token = s['DAI_balance'] / s['ETH_balance']
+    real_Token = uniswap_events['token_balance'][data_counter] / uniswap_events['eth_balance'][data_counter]
+    model_ETH = s['ETH_balance'] / s['DAI_balance']
+    real_ETH = uniswap_events['eth_balance'][data_counter] / uniswap_events['token_balance'][data_counter]
```

Comment:
I think that you can declare these variables inside the if condition section that they belong. This could make your code cleaner, and separate better the variables.